server/routes.go

```diff
@@ -747,6 +776,9 @@ func Serve(ln net.Listener, allowOrigins []string) error {
 	defer os.RemoveAll(workDir)
 
 	r := gin.Default()
+
+	r.Use(RequestLogger())
```

Comment:
To avoid logging a bunch of info by default I'd prefer to see this logged only in the case that an environment variable is set.

Maybe something like `OLLAMA_LOG_LEVEL`, so we can use the same env var to set levelled logging in the future:
```
var logLevel = strings.ToLower(os.Getenv("OLLAMA_LOG_LEVEL"))
if logLevel == "debug" {
   r.Use(RequestLogger())
}
```

Reply:
@BruceMacD added environment variable and updated README.md with all environment variables currently in Ollama.